src/rapidfire.cpp

```diff
@@ -53,6 +61,8 @@ Rapidfire::SendIndexCmd(uint8_t index)
     uint8_t newBand = index / 8 + 1;
     uint8_t newChannel = index % 8;
 
+    if (!SPIModeEnabled) EnableSPIMode();
```

Comment:
This should be in SendSPI() so that it is checked if a separate command is sent like the buzzer.